required_fields.node.js

```diff
@@ -25,7 +25,9 @@ module.exports = function (homeValues) {
         wall_construction_same : mandatoryMessage,
         window_construction_same : mandatoryMessage
     };
-
+    
+    const trueOptions = [1, '1', true];
+    const falseOptions = [0, '0', false];
```

Comment:
I like this solution. It makes our validation engine less mean. Two things I'd change, though:

Instead of calling indexOf() everywhere, I'd suggest simplifying by just checking once:

```
const isBlowerDoorTestTrue = homeValues['blower_door_test'].indexOf([1, '1', true, 'true']) > 0;
const isBlowerDoorTestFalse = homeValues['blower_door_test'].indexOf([0, '0', false, 'false']) > 0;
```

I also suspect we could save ourselves trouble in the future by putting this behavior into a generic `castBool()` method that returns TRUE, FALSE, or NULL, and using it to cast all of our boolean values.

Also, as I've done above, I'd add the strings "true" and "false" since there's no reason we couldn't accept them and it can be horrifically hard to figure out why `if ('false')` never executes :)

Reply:
This isn't just for blower doors.  It's for any of our boolean radios (blower door, construction the same, etc).

Reply:
I realize that. Is there a reason not to have it in a generic boolean casting function and use it for all boolean values in the home? The fact that certain values are radio buttons in our GUI shouldn't influence how we validate them. I think it would be ideal if we treated all boolean values identically. And having a function call with a nice name would also be more readable than repeatedly checking whether a value is in an array with indexOf()